src/components/verification-grid/verification-grid.ts

```diff
@@ -866,72 +891,33 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
 
   private async handlePreviousPageClick(): Promise<void> {
     if (this.canNavigatePrevious()) {
-      this.historyHead += this.populatedTileCount;
-      await this.renderHistory(this.historyHead);
+      this.pageBackward();
     }
   }
 
   private handleNextPageClick(): void {
-    if (this.isViewingHistory()) {
-      this.pageForwardHistory();
-    }
-  }
-
-  private async pageForwardHistory(): Promise<void> {
     if (this.canNavigateNext()) {
-      this.historyHead -= this.populatedTileCount;
-      await this.renderHistory(this.historyHead);
+      this.pageForward();
     }
   }
 
-  private async renderHistory(historyOffset: number) {
-    const decisionStart = Math.max(0, this.subjectHistory.length - historyOffset);
-    const decisionEnd = Math.min(this.subjectHistory.length, decisionStart + this.populatedTileCount);
-    const decisionHistory = this.subjectHistory.slice(decisionStart, decisionEnd);
+  private async pageForward(): Promise<void> {
+    const proposedViewHead = this.viewHead + this.populatedTileCount;
+    this.viewHead += Math.min(proposedViewHead, this.verificationHead);
```

Comment:
first question is why? and i think the reasoning here is so that if the grid size changed while viewing history, when you get back to verification head, it will "align" again so that there's a full set of undecided items on the page

but we don't need to do that anymore.

if we page forward and the verification head is in the middle of a page, it shouldn't matter. `N` tiles before the verification head will already be done, and `M` afterwards will need to be done, but the logic for the entire set `P` is  'do all items have all actions completed?' 

While typing this out, I thought about a counter argument: then a user might have to sub-select to make new decisions for just the `N` subset.... 

I don't know. The intent here is to make this feel like a scrollable list. I think this is inconsistent.